src/components/navbar.tsx

```diff
@@ -7,7 +7,7 @@ export type NavbarProps = HTMLAttributes<HTMLElement>;
 export const Navbar = ({ className, ...props }: NavbarProps) => (
   <nav
     className={clsx(
-      "fixed top-0 left-0 right-0 z-navbar flex gap-8 bg-black/50 px-4 backdrop-blur-xl",
+      "fixed top-0 left-0 py-0 right-0 z-navbar flex justify-between gap-8 bg-black/50 px-2 backdrop-blur-xl",
```

Comment:
Let's keep the `px-4` padding - menu button being so close to the edge of a screen can be harder to tap on phones with bent screens on the side. Also `py-0` can be removed as there is no vertical padding by default.

```suggestion
      "fixed top-0 left-0 right-0 z-navbar flex justify-between gap-8 bg-black/50 px-4 backdrop-blur-xl",
```